.github/workflows/pr_acceptance.yml

```diff
@@ -5,8 +5,8 @@ on:
     branches:
       # A test branch for seeing if your tests will pass in your personal fork
       - test_me_github
-  pull_request_target:
-    types: [opened, reopened, synchronize]
+    #  pull_request_target:
```

Comment:
Can we uncomment this and try changing `ubuntu-latest` to `ubuntu-18.04` below?

It seems to be a bug in some of the underlying hosts and I've filed an [associated bug](https://github.community/t/podman-issues-on-ubuntu-latest/182129).